### ISSUE TITLE
PSM folder: the three Info fields on the PSM folder whose titles begin with "Historic Designation Ap...

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#11932]
> What application are you using?

AMANDA

> Describe the problem.

PSM folder: the three Info fields on the PSM folder whose titles begin with "Historic Designation Approval" should be consolidated into a single Info field titled "Located in Local or National Historic District?"

> How soon do you need this?

Flexible  — An extended timeline is OK

> Requested By
Kyle K.

Request ID: DTS23-106950

**Comment by vschandramouli** [comment]
Updated AC, moving to Backlog.

**Comment by vschandramouli** [comment]
SMEs mentioned there may be a portal component to this, including potential addition of help text. Moving back to Needs Scoping.

**Comment by TracyLinder** [comment]
Will do portal help text on other issue, moving to backlog

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Committed Changes and requested for Dev deployment - SCTASK0313754

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Changes deployed to Test, Ready for UAT

**Comment by Nadin-Nader** [comment]
Sent a message to Kyle and ROW to please UAT this on TEST, Waiting on Response

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT Kyle UATd this and needs the following things added (Let me know if this is a quick addition or if it should go on another card):

1. Can this be marked as a required field (red asterisk)?
2. Can we also add a link to GIS so folks can look up Historic Districts?
3. Can this field be moved down to be in the same area as the Walk of Stars fields, as shown in the below screenshot? This is where the Historic fields used to be and it makes the most sense in that spot.
![MicrosoftTeams-image (42).png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/d5f0ca4d-7629-40f5-b407-805a47ebc9b2)

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created request for Dev and Test Deployment 
SCTASK0323356

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been deployed to Dev and TEST. please UAT and let me know if it is ready for Stake holder UAT

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader committed the change to add a text change in helptext and requested for Deployment
SCTASK0325304

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT This has been deployed. Please UAT

**Comment by Lpriya-LIT** [comment]
![image.png](https://images.zenhubusercontent.com/611eac1954217f265acc0499/626b0043-b785-416c-ad90-1d46b3319582)

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Ready for UAT on Test

**Comment by Nadin-Nader** [comment]
Sent message to Kyle asking him to UAT this on TEST

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT here is Kyles response to this one:

[11:22 AM] Kampe, Kyle

11932: We need this to be a required field for the external customer, AKA "Mandatory for Initial Set up".

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT here is Kyles response to this one:

[11:22 AM] Kampe, Kyle

11932: We need this to be a required field for the external customer, AKA "Mandatory for Initial Set up".

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been approved by Kyle to get deployed to PROD

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Changes Promoted to Prod, Ready for UAT

**Comment by Nadin-Nader** [comment]
[3:23 PM] K, Kyle
   
11932 and 11972: Rejected. I'm not seeing the new Historic field or the associated help text when applying on the AB+C portal:

![MicrosoftTeams-image (46).png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/65f59efe-c167-4309-bdc5-715d40e4a1cf)

The "Located in Local or National Historic District" is visible in the AMANDA Info tab, but apparently not in AB+C when applying for a new PSM folder.

**Comment by Nadin-Nader** [comment]
Closing as Kyle has approved this on PROD
